ansible_ai_connect_chatbot/README.md

```diff
@@ -0,0 +1,69 @@
+# Ansible AI Connect Chatbot
+
+## Build from source
+
+To start building the "Admin Portal" project, you're going to need:
```

Comment:
Copy/paste mistake "... building the 'Chat Bot'..."

Reply:
I have to confess that I was not reading the text.  Thank you, I will fix that.